app/models/restaurant.rb

```diff
@@ -0,0 +1,2 @@
+class Restaurant < ApplicationRecord
+end
```

Comment:
I would make an association to RestaurantCategory and validate presence

Reply:
We were sticking to the story, and its about going from main page to restaurant page.

Reply:
😎